t/50logconf.t

```diff
@@ -0,0 +1,176 @@
+# Set TEST_DEBUG=1 to dump access logs
+
+use strict;
+use warnings;
+use Net::EmptyPort qw(empty_port wait_port check_port);
+use File::Temp qw(tempdir);
+use JSON;
+use Time::HiRes qw(sleep);
+use Test::More;
+use t::Util;
+
+my $client_prog = bindir() . "/h2o-httpclient";
+plan skip_all => "$client_prog not found"
+    unless -e $client_prog;
+plan skip_all => 'mruby support is off'
```

Comment:
is ruby support required? I was able to run this test without it

Reply:
Oops, removed it! Thanks.

Feel free to merge this PR to yours, https://github.com/h2o/h2o/pull/2973.